h2o-algos/src/main/java/hex/StackedEnsembleModel.java

```diff
@@ -59,13 +61,18 @@ public StackedEnsembleModel(Key selfKey, StackedEnsembleParameters parms, Stacke
 
     // base_models is a list of base model keys to ensemble (must have been cross-validated)
     public Key<Model> _base_models[] = new Key[0];
-    // Should we keep the level-one frame of cv preds + repsonse col?
+    // Should we keep the level-one frame of cv preds + response col?
     public boolean _keep_levelone_frame = false;
+    // internal flag if we want to avoid having the base model predictions rescored multiple times, esp. for blending.
+    public boolean _keep_base_model_predictions = false;
```

Comment:
For now, I chose not to expose this field through REST API, so there's no possibility for user to enable predictions caching.
Main intent in mind is to use this for SEs in AutoML as we compute multiple SEs reusing same models.

Reply:
Is this meant for grid search over SE hyper-parameters?

Reply:
It's not what I had in mind: just that AutoML currently builds 2 SEs, one from all models, and the other one from bests of kind, therefore reusing same models in each SE, besides when running the AutoML instance multiple times, new SEs need to be built, including previous models again. This flag could prevent having to recompute predictions in those cases.

But if we would want to do grid search over SE hyper-params, this flag would provide similar benefits.

Reply:
Does AutoML delete the cached predictions? I couldn't find it in the code. I only see it deleted when the SE model is deleted, the base predictions can be quite big I think to keep around without the user knowing. It is dense numerical data that do not compress well.